content/credits/_index.es.md

```diff
@@ -1,26 +1,29 @@
 ---
-title: "Creditos"
+title: "Créditos"
 weight: 200
 ---
 
 # Patrocinado por
 
 [![efabless](/images/efabless.png)](https://efabless.com/)
 
+[![aisler](/images/aisler.png)](https://aisler.net/)
+
 # Equipo
 
 Tiny Tapeout no sería posible sin la ayuda de muchas personas. Nos gustaría agradecer especialmente a:
 
 * Uri Shaked por el desarrollo de [wokwi](https://wokwi.com/) y mucho más
 * [Sylvain Munaut](https://twitter.com/tnt) por ayudar con mejoras en la cadena de escaneo
 * [Mike Thompson](https://www.linkedin.com/in/michael-thompson-0a581a/) por su experticia en verificación
-* [Jix](https://twitter.com/jix_) por soporte de verificación formal
+* [Jix](https://twitter.com/jix_) por soporte formal de verificación
```

Comment:
In English it's 'formal verification', a type of verification done with SAT solvers. In Spanish it's probably as it was before 'verificacion formal'